sequencer/src/header.rs

```diff
@@ -251,14 +248,15 @@ impl BlockHeader for Header {
         )
     }
 
+    // TODO return metadata must be cloned from returned Payload; don't return metadata??
```

Comment:
Yeah, we could remove this and have HotShot get the metadata by calling `payload.metadata()`. But I feel like it's going to end up cloning the metadata anyways. We could make metadata an `Arc<NamespaceTable>` if we're really concerned. But we're also assuming metadata is small